html_app/model/Treatment.js

```diff
@@ -73,25 +73,34 @@ scb.Treatment = function scb_Treatment(data, context, parent) {
 	Object.defineProperty(self, 'duration', {
 		get : function() {
 			var time = parseFloat(data.duration_value);
-			//var days = Math.floor(time / 86400);
 			var days = Math.floor((time % 604800) / 86400);
-
 			var hours = Math.floor((time % 86400) / 3600);
-			var minutes = Math.round((time % 3600) / 60);
-			
+			var minutes = Math.floor((time % 3600) / 60);
```

Comment:
nitpick: seems like there should be a builtin function for this (you can chose to ignore "nitpicks")